views/js/upload.js

```diff
@@ -31,6 +45,13 @@ uploadButton.addEventListener('click', function () {
 		const formData = new FormData();
 		formData.append('file', file);
 
+		axios
+	.post(`${baseURL}/a`,)
+			.then(( res ) => {
+				console.log('upload.js In / axios.post / res : ', res);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We detected that unsanitized user data is being written to the logs. Unsanitized data can inject malicious content. To increase the security of your code, sanitize your inputs before logging them. 

[Learn more](https://cwe.mitre.org/data/definitions/117.html)